cloudsync/providers/filesystem.py

```diff
@@ -319,49 +319,56 @@ def __init__(self, namespace="/"):
         super().__init__()
 
     @property
-    def namespace(self):
+    def namespace(self) -> Namespace:
```

Comment:
Should this be `Optional`? It's typed as that implicitly in the constructor, and there are runtime checks in a couple places

Reply:
Definitely, good catch. Fixed for Filesystem and Mock providers.